lib/hardware/memories/ram/iob_ram_tdp_be/iob_ram_tdp_be.py

```diff
@@ -37,6 +36,30 @@ def setup(py_params_dict):
                 "max": "1",
                 "descr": "No simultaneous read/write",
             },
+            {
+                "name": "COL_W",
+                "type": "F",
+                "val": "DATA_W / 4",
+                "min": "NA",
+                "max": "NA",
+                "descr": "",
+            },
+            {
+                "name": "NUM_COL",
+                "type": "F",
+                "val": "DATA_W / COL_W",
+                "min": "NA",
+                "max": "NA",
+                "descr": "",
+            },
+            {
+                "name": "file_suffix",
```

Comment:
```suggestion
                "name": "FILE_SUFFIX",
```
Parameter should be uppercase like in snippet: https://github.com/IObundle/iob-soc/pull/943/files/a27903b036def55f4141eee63eb10946e0612ab9..f1f2f4c5bf0214a9e65f172ca6d62f7e9d273c46#diff-91b856d007356aa86fb6859fdfba330adb972f2d8b87f4951351fa99669272b1R108